### PR TITLE
Fixed urllib3 instrumentation example in instrumentation documentation

### PR DESCRIPTION
**Pull request by yashaswi2000**
# Description
The usage example mentioned in the docs: https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/urllib3/urllib3.html has a syntax issue, where the correct syntax is 
```
URLLib3Instrumentor().instrument(
request_hook=request_hook, response_hook=response_hook)
)
```
instead of 
```
URLLib3Instrumentor.instrument(
request_hook=request_hook, response_hook=response_hook)
)
```

Fixes ([#1767](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1767))

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

tested by regenerating the docs, using tox.

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [x] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [x] Documentation has been updated

### CODE REVIEW

**Review by shalevr — APPROVED** [review]
Thank you

**Comment by yashaswi2000** [comment]
do i need to author a commit again?, i signed the easycla now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fixed urllib3 instrumentation example in instrumentation documentation
```
Comment (shalevr) [inline CHANGELOG.md:36]: Please move this entry to the Unreleased section

Comment (ocelotl) [reply CHANGELOG.md:41]: Actually, we don't need a changelog entry for this PR.

**Pull request merged**